src/test/kotlin/com/atlassian/performance/tools/hardware/report/HardwareExplorationChartTest.kt

```diff
@@ -72,7 +72,7 @@ class HardwareExplorationChartTest {
     @Test
     fun shouldPlotDbRecommendation() {
```

Comment:
The chart title is hardcoded to `Jira XL profile hardware exploration`

Reply:
In general the `hardware-recommendation-chart-template.html` is full of garbage, like chart-js code, Chrome extension code, hardcoded JPT commit etc.
Please clean it up, best to start from scratch: `hardware-exploration-chart-template.html`

Reply:
yep, will do